zulipterminal/ui_tools/views.py

```diff
@@ -1535,31 +1537,37 @@ def __init__(
                     ("Date & Time", date_and_time),
                     ("Sender", msg["sender_full_name"]),
                     ("Sender's Email ID", msg["sender_email"]),
-                    (
-                        "View message in browser",
-                        f"Press {view_in_browser_keys} to view message in browser",
-                    ),
-                    (
-                        "Full rendered message",
-                        f"Press {full_rendered_message_keys} to view",
-                    ),
-                    (
-                        "Full raw message",
-                        f"Press {full_raw_message_keys} to view",
-                    ),
                 ],
             ),
         ]
+
+        # actions for message info popup
+        viewing_actions = (
+            "Viewing Actions",
+            [
+                (
+                    "Open in web browser",
+                    view_in_browser_keys,
```

Comment:
If you remove the commas at the end of some of these lines, `black` may be allow them all on one line.